SDALib-CPP/SDAReplay/ReplayMain.cpp

```diff
@@ -1,11 +1,41 @@
 #include "ReplayDriver.h"
+#include <boost/program_options.hpp>
 
-int main()
+using namespace boost::program_options;
```

Comment:
do not use this but use somthing like 'namespace poptions = boost::program_options'

Reply:
I did try something like this, only tried it with #define which didn't work, pushed a commit with your request